invenio_vocabularies/contrib/names/datastreams.py

```diff
@@ -48,7 +49,7 @@ def _fetch_orcid_data(self, orcid_to_sync, bucket):
         key = f"{suffix}/{orcid_to_sync}.xml"
         try:
             return self.s3_client.read_file(f"s3://{bucket}/{key}")
-        except Exception as e:
+        except Exception:
             # TODO: log
             return None
```

Comment:
Nothing to do with this PR, I think this is my leftover I will create a PR to fix this it shouldn't silently fail

Reply:
I've added `current_app.logger.exception(...)`, so we don't have a TODO for now. But yeah, if you know in what ways this can fail, it might be good to handle in a special way.